repos.conf.d/ubuntu/repo.conf

```diff
@@ -0,0 +1,2 @@
+repo_url="rsync://archive.ubuntu.com/ubuntu/"
+repo_provider="rsync"
```

Comment:
Yes. I was hoping to not make any changes to the repo filter at the same time we broke out the configs from the main repo. I intentionally submitted the PR with no updates since it will be hard to follow them with no revision history.